src/music/tonation.py

```diff
@@ -3,6 +3,18 @@
 
 
 class Tonation(base_sound.BaseSound):
+    """A sound of one pitch present in a music piece
+
+    Attributes:
+        kind (str) : "minor" or "major"
+        tonic (Chord) : Set of sounds composing the
```

Comment:
I feel like we can delete toniv, subdominant and dominant params. It is only used in bar_algorithm and we can remove it (or even leave just daria)

Reply:
Removed